include/rxmesh/reduce_handle.h

```diff
@@ -119,8 +119,37 @@ class ReduceHandle
         return std::sqrt(reduce_2nd_stage(stream, cub::Sum(), 0));
     }
 
+    //does either arg max or min depending on second argument is_min
+    T arg_maxOrmin(const Attribute<T, HandleT>& attr,
+              bool                         is_min       = true,
+              uint32_t                     attribute_id = INVALID32,
```

Comment:
Also, we don't need to specify the `attribute_id` since we can only do `arg_max`/`arg_min` on a scalar Attributes.